src/ansible_navigator/actions/settings.py

```diff
@@ -0,0 +1,201 @@
+"""The ``settings`` subcommand action."""
+
+import curses
+
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
+
+from .._yaml import human_dump
+from ..action_base import ActionBase
+from ..action_defs import RunStdoutReturn
+from ..app_public import AppPublic
+from ..configuration_subsystem import transform_settings
+from ..steps import Step
+from ..ui_framework import CursesLinePart
+from ..ui_framework import CursesLines
+from ..ui_framework import Interaction
+from . import _actions as actions
+from . import run_action
+
+
+def filter_content_keys(obj: Dict[Any, Any]) -> Dict[Any, Any]:
+    """Filter out some keys when showing content.
+
+    :param obj: The object to be filtered
+    :return: The object with keys filtered out
+    """
+    return {k: v for k, v in obj.items() if not k.startswith("__")}
+
+
+def color_menu(colno: int, colname: str, entry: Dict[str, Any]) -> Tuple[int, int]:
+    # pylint: disable=unused-argument
+    """Color the menu.
+
+    :param colno: Column number
+    :param colname: Column name
+    :param entry: Column value
+    :return: Tuple int used to color the menu
+    """
+    if entry["is_default"]:
+        return 2, 0
+    return 3, 0
+
+
+def content_heading(obj: Any, screen_w: int) -> CursesLines:
+    """Create a heading for the setting entry showing.
+
+    :param obj: The content going to be shown
+    :param screen_w: The current screen width
+    :return: The heading
+    """
+    heading = []
+    string = obj["name"].replace("_", " ")
```

Comment:
What is the meaning of this `string` thingy? Is it the whole setting line? Just the name? Why isn't it called `setting_name` or something like that, then? If the purpose/content of a variable is unobvious without having to stick a debugger in this place, it could probably be improved by making the context clear via renaming.

Reply:
I updated this to "heading_text". 
Example snippet:
```
heading_text = obj["name"].replace("_", " ")
    if obj["is_default"]:
        heading_text += f" (current/default: {obj['current_value']})"
        color = Color.GREEN
```